### PR TITLE
Homework one - Ismail Ahmadi

### PR DESCRIPTION
**Pull request by Ismail-ahmadi**
🚨 REMEMBER

1️⃣ We will review together the work you have done so far. 

2️⃣ Add my name in the Reviewers section in the right sidebar 👉👉👉

3️⃣ Answer these reflection questions before each PR:

  * What was most challenging about the homework so far? 

  * How did you solve it?

### CODE REVIEW

File: terminal.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### No 5
+- cd ../
+- cd ../../
+- cd ../work/project_to_delegate
```
Comment (nikosolihin) [inline terminal.md:4]: Nice move!

File: terminal.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### No 5
+- cd ../
+- cd ../../
+- cd ../work/project_to_delegate
+- cd ../../home
+
+### No 10
+- rm random.txt
+- rm -rf home
+- cd ../ rm -rf home
```
Comment (nikosolihin) [inline terminal.md:10]: I know you meant to put these two commands on separate lines.

But if you want to execute a one liner, here's a tip: `cd .. && rm -rf home`.

Chain as many commands separated with `&&`

**Review by nikosolihin — CHANGES_REQUESTED** [review]
So far so good @Ismail-ahmadi. 

Just missing `forking.md`, `github.md` and reflection questions.

File: terminal.md (hunk 1 above)
Comment (Ismail-ahmadi) [reply terminal.md:4]: Thanks

File: terminal.md (hunk 2 above)
Comment (Ismail-ahmadi) [reply terminal.md:10]: yeah, thanks for your feedback and I've done the rest of the homework

**Pull request merged**